docs_src/general/Definitions.md

```diff
@@ -0,0 +1,62 @@
+# Definitions
+The following glossary provides terms used in EdgeX Foundry.  The definition are based on how EdgeX and its community use the term versus any strict technical or industry definition.
+
+<a name="browngreenfield"></a>
```

Comment:
Why do you have anchor tags? These are unnecessary

Reply:
fixed